test-cases/longevity/longevity-sla-system-24h.yaml

```diff
@@ -0,0 +1,29 @@
+test_duration: 1500
+
```

Comment:
IIUC there are "hidden" stress threads that will happen during the test as part of the SlaNemesis.
I think it should be described here somewhere.

Reply:
If we create those by code, what's the point of listing them here ? 
we don't list other places we are using stress command for short duration (i.e. in other nemesis add_dc for example, or others that uses s-b)

We list the main load in the test case, but not every single call to stress tools is in the configuration.

Reply:
listing all of them in configuration, leads to extra configuration we use only in specific nemesis or specific test case.
you'll need to configure it for all of the tests, if you'll want those nemesis to run across all tests.

I think we should turn to commands in configuration, only when it's needed.

it's clear enough in logs and events which one were used, we can't put all the information in configuration

Reply:
I didn't mean to list them.
I meant to add a comment saying "this is the main thread that is / isn't set with SLA" (is it?).
"During the test we start X (2?) more (decided by parameter BLA)"
...

Reply:
It's not very clear to me, and I know many of the details + I reviewed the PR.

Reply:
Added a comment:
```
# This is the main stress thread that will run with default role (cassandra)
```